src/databricks/sql/client.py

```diff
@@ -152,8 +153,13 @@ def read(self) -> Optional[OAuthToken]:
                             experimental_oauth_persistence=DevOnlyFilePersistence("~/dev-oauth.json")
                         )
                 ```
-
-
+            :param _use_arrow_native_complex_types: `bool`, optional
+                Controls whether a complex type field value is returned as a string or as a native Arrow type. Defaults to True.
+                When True:
+                    MAP is returned as List[Tuple[str, Any]]
```

Comment:
Why is MAP not returned as a Dict[str, any]?

Reply:
I had the same thought but this seems to be controlled by `pyarrow`. Updating to use a `dict()` is a potential improvement we could implement.

Reply:
Unclear if it would be an improvement.  If this is the native type, let's leave it be.